pkgs/top-level/all-packages.nix

```diff
@@ -5823,6 +5826,8 @@ in
 
   mcfly = callPackage ../tools/misc/mcfly { };
 
+  m2r = python3Packages.callPackage ../tools/text/m2r { };
```

Comment:
Avoid using the `callPackage` from another scope. Instead, pass in `python3` or `python3Packages`.

Reply:
Oh man.

This is a tool written in Python, using Python tools as dependencies and even using Python traditional tools for packaging. This is not a typical "gcc-configure-make" app.

I can't see why not to use `python3Packages.callPackage` here. Is there some deprecation I am not aware?

Reply:
IIRC if you add python to inputs it is a different python which is very hard to debug.

Reply:
This is also standard in QT packages. I still remember when I updated openshot.